viewing_party/main.py

```diff
@@ -0,0 +1,135 @@
+# wave 1
+def create_movie(movie_title, genre, rating):
+    new_movie = {}
+    if movie_title == None or genre == None or rating == None:
+        new_movie = None
+    else:
```

Comment:
you could also rewrite to build the dictionary like so

new_move = {
                        "title": movie_title,
                        "genre": genre, 
                          "rating": rating
                       }